kernels/simple_matmul/main.c

```diff
@@ -21,8 +21,9 @@
  * /target/snitch_cluster/sw/snax/mac/include"
  *
  * */
-#include "snax-gemm-lib.h"
```

Comment:
Can we not include these routines now, instead of copy-pasting them?

Reply:
what exactly would you like included in a separate routine? most of this comes straight out of the template